tests/test_filters.py

```diff
@@ -0,0 +1,184 @@
+    assert filter['operator'] == 'greaterThan'
+    assert filter['args'][0] == {
+        'type': 'field',
+        'value': '[-hub-][field][[[ems-core][entity-type][foqa-flights]][[ems-core][base-field][flight.uid]]]'
+
+    }
+    assert filter['args'][1] == {
+        'type': 'constant',
+        'value': '17000'
+    }
+
+def test_greaterThanOrEqual():
+    query = MockFilterQuery('Flight Record')
+    query.filter("'Flight Record' >= '17000'")
+    filter = get_filter(query)
+    assert filter['operator'] == 'greaterThanOrEqual'
+    assert filter['args'][0] == {
+        'type': 'field',
+        'value': '[-hub-][field][[[ems-core][entity-type][foqa-flights]][[ems-core][base-field][flight.uid]]]'
+
+    }
+    assert filter['args'][1] == {
+        'type': 'constant',
+        'value': '17000'
+    }
+
+def test_lessThan():
+    query = MockFilterQuery('Flight Record')
+    query.filter("'Flight Record' < '17000'")
+    filter = get_filter(query)
+    assert filter['operator'] == 'lessThan'
+    assert filter['args'][0] == {
+        'type': 'field',
+        'value': '[-hub-][field][[[ems-core][entity-type][foqa-flights]][[ems-core][base-field][flight.uid]]]'
+
+    }
+    assert filter['args'][1] == {
+        'type': 'constant',
+        'value': '17000'
+    }
+
+def test_lessThanOrEqual():
+    query = MockFilterQuery('Flight Record')
+    query.filter("'Flight Record' <= '17000'")
+    filter = get_filter(query)
+    assert filter['operator'] == 'lessThanOrEqual'
+    assert filter['args'][0] == {
+        'type': 'field',
+        'value': '[-hub-][field][[[ems-core][entity-type][foqa-flights]][[ems-core][base-field][flight.uid]]]'
+
+    }
+    assert filter['args'][1] == {
+        'type': 'constant',
+        'value': '17000'
+    }
+
+def test_notBetweenExclusive():
+    query = MockFilterQuery('Flight Record')
+    query.filter("'15000' > 'Flight Record' > '17000'")
+    filter = get_filter(query)
+    assert filter['operator'] == 'notBetweenExclusive'
+    assert filter['args'][0] == {
+        'type': 'field',
+        'value': '[-hub-][field][[[ems-core][entity-type][foqa-flights]][[ems-core][base-field][flight.uid]]]'
+
+    }
+    assert filter['args'][1] == {
+        'type': 'constant',
+        'value': '15000'
+    }
+    assert filter['args'][2] == {
+        'type': 'constant',
+        'value': '17000'
+    }
+
+def test_betweenExclusive():
+    query = MockFilterQuery('Flight Record')
+    query.filter("'15000' < 'Flight Record' < '17000'")
+    filter = get_filter(query)
+    assert filter['operator'] == 'betweenExclusive'
+    assert filter['args'][0] == {
+        'type': 'field',
+        'value': '[-hub-][field][[[ems-core][entity-type][foqa-flights]][[ems-core][base-field][flight.uid]]]'
+
+    }
+    assert filter['args'][1] == {
+        'type': 'constant',
+        'value': '15000'
+    }
+    assert filter['args'][2] == {
+        'type': 'constant',
+        'value': '17000'
+    }
```

Comment:
Regarding a previous comment, can you add tests for datetime filters?  I ask because they respect different operators.

Reply:
I added tests on c30d338